.github/workflows/multimodal-pr.yml

```diff
@@ -107,3 +100,41 @@ jobs:
           ipython kernel install --name "python3" --user
           echo "Current branch: ${BRANCH_NAME}"
           papermill --parameters repo_branch ${BRANCH_NAME} notebooks/phrase_grounding.ipynb /tmp/phrase_grounding_output.ipynb
+
+  publish:
+    runs-on: ubuntu-18.04
+    needs: [ flake8, mypy, pytest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Install pypa/build
+        run: pip install build
+
+      - name: Build a binary wheel and a source tarball
+        run: python -m build --sdist --wheel --outdir ${{ env.folder }}/dist/ ${{ env.folder }}
+
+      - name: Publish package to TestPyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TEST_API_TOKEN }}
+          packages_dir: ${{ env.folder }}/dist/
+          repository_url: https://test.pypi.org/legacy/
+          verbose: true
+          skip_existing: true
+
+      - name: Publish package to PyPI
+        if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
@ant0nsc This is not good, right? It looks like it will try to upload a new version to PyPI when a commit is tagged, in general. But we don't necessarily want to publish to PyPI all packages just because we have a tag. Is that correct?

Reply:
Yes, that is precisely the strategy we have been using so far: Each tag triggers publishing a package, and the tag value becomes the package version.

Reply:
I guess I'm just a bit confused about the sequence of events. For example, say there is a tagged commit that includes a change to a file in `hi-ml-multimodal`. The tag is for, e.g., a new version of `hi-ml-azure`.

Will the publishing actions for `hi-ml` and `hi-ml-multimodal` also be triggered?